jobs/JEVS_MESOSCALE_PLOTS

```diff
@@ -99,19 +99,20 @@ fi
 ####################################
 # Define COMIN/COMOUT variables:
 ####################################
-export COMIN=${COMIN:-$(compath.py -o $NET/$evs_ver)}
-export COMINapcp24mean=${COMINapcp24mean:-$(compath.py $envir/com/$NET/$evs_ver)/stats/$COMPONENT}
+export EVSINapcp24mean=${EVSINapcp24mean:-$(compath.py $envir/com/$NET/$evs_ver)/stats/$COMPONENT}
+export EVSINccpa=${EVSINccpa:-$(compath.py -o $COMIN/prep/${COMPONENT})/${RUN}}
+export EVSINmrms=${EVSINmrms:-$(compath.py -o $COMIN/prep/${COMPONENT})/${RUN}}
+export EVSINspcotlk=${EVSINspcotlk:-$(compath.py -o $COMIN/prep/${COMPONENT})/${RUN}}
```

Comment:
We are getting closer, @RoshanShrestha-NOAA ! lines 103-105 still need some modifications. Since we are using $COMIN now, you don't need to use compath. Please update lines 103-105 to look like this:
```
export EVSINccpa=${EVSINccpa:-$COMIN/prep/${COMPONENT}/${RUN}}
export EVSINmrms=${EVSINmrms:-$COMIN/prep/${COMPONENT}/${RUN}}
export EVSINspcotlk=${EVSINspcotlk:-$COMIN/prep/${COMPONENT}/${RUN}}
```

Reply:
I forgot line 102, but it also applies to this comment. Please follow the same pattern.